docs/streamingapi/code-snippets/detect-key-phrases.md

```diff
@@ -257,6 +257,9 @@ ws.onopen = (event) => {
   }));
 };
 ```
+:::note Backward Compatibility
+The previous endpoint  `wss://api.symbl.ai/v1/realtime/insights/` is now updated to `wss://api.symbl.ai/v1/streaming/` to standardize our API nomenclature. This change is backward compatible; however, we recommend you to use the new endpoint. 
```

Comment:
This semicolon is not required here. Keep the docs simple with minimal punctuations. Add a full stop after compatible and start the sentence with "However, we ...."